plugin/core/views.py

```diff
@@ -48,6 +49,21 @@ def region_to_range(view: sublime.View, region: sublime.Region) -> Range:
     )
 
 
+def encoded_filename(uri: str, position: Dict[str, Any]) -> str:
+    # WARNING: Cannot possibly do UTF-16 conversion :) Oh well.
+    return '{}:{}:{}'.format(uri_to_filename(uri), position['line'] + 1, position['character'] + 1)
```

Comment:
Style nit: Since it's not reused and used only in one place maybe just inline it?